packages/toolkit/src/view/hub/hub-view/card-pipeline-hub/Head.tsx

```diff
@@ -40,23 +40,39 @@ export const Head = ({
   }, [pipeline]);
 
   const displayName = React.useMemo(() => {
-    const owner = pipeline.owner as UserOwner;
-    const userID = owner?.user?.name?.split("/")[1]; // Get the part after "users/"
-    const ownerDisplayName =
-      owner?.user?.profile?.company_name ||
-      owner?.user?.profile?.display_name ||
-      "";
-
-    if (userID) {
-      return userID;
-    } else {
+    const owner = pipeline.owner;
+  
+    if ('user' in owner) {
+      // Owner is of type user
+      const userOwner = owner as UserOwner;
+      const userID = userOwner.user.name.split("/")[1]; // Get the part after "/"
+      const ownerDisplayName =
+        userOwner.user.profile?.display_name ||
+        userID;
+  
       // If ownerName contains spaces, return it in lowercase with hyphens
       if (ownerDisplayName.includes(" ")) {
         return ownerDisplayName.toLowerCase().replace(/\s+/g, "-");
       } else {
         // If the owner name doesn't contain spaces, return it in lowercase
         return ownerDisplayName.toLowerCase();
       }
+    } else if ('organization' in owner) {
+      // Owner is of type organization
+      const orgOwner = owner as OrganizationOwner;
+      const orgID = orgOwner.organization.name.split("/")[1]; // Get the part after "/"
```

Comment:
```suggestion
      const orgID = orgOwner.organization.id
```

@thewbuk You can directly get the id